src/TRestTrackAnalysisProcess.cxx

```diff
@@ -41,11 +41,11 @@
 ///
 /// Maximum Second Track Energy observables:
 ///
-/// * **SecondTrackMaxEnergy**: Energy of the second most energetic track in the event with X,Y,Z
+/// * **tckSecondMaxEnergy**: Energy of the second most energetic track in the event with X,Y,Z
```

Comment:
why to rename the observable?

Reply:
For coherence with the already existing name `MaxTrackEnergy`.  I thought it was better like this. Also, the observables regarding the second track were sometimes named `secondTrackMax...` and other times `secondMaxTrack...`. So I just renamed everything following the same style as for the one track observables.

Reply:
Ah, this is probably an error becuase I replaced the name automatically. The new name should be `SecondMaxTrackEnergy`

Reply:
I've corrected this in the next commit (587d68e)

Reply:
Ok, I see, sorry I did not catch these messages on time.